src/scene/shader-lib/chunks/lit/vert/normal.js

```diff
@@ -30,9 +30,16 @@ vec3 getNormal() {
     #endif
 
     #ifdef MORPHING_TEXTURE_BASED_NORMAL
+
+    #ifdef WEBGPU
+            ivec2 morphUV = getTextureMorphCoords();
+            vec3 morphNormal = texelFetch(morphNormalTex, ivec2(morphUV), 0).xyz;
+        #else
+            vec2 morphUV = getTextureMorphCoords();
+            vec3 morphNormal = texture2D(morphNormalTex, morphUV).xyz;
+        #endif
```

Comment:
Indentation here looks inconsistent.

Reply:
good point: Fixed incorrect indentation in normal.js chunk
thanks.